include/pulsar/Result.h

```diff
@@ -94,6 +94,7 @@ enum Result
     ResultInterrupted,  /// Interrupted while waiting to dequeue
 
     ResultDisconnected,  /// Client connection has been disconnected
+    ResultNotFound       /// The generic was not found
```

Comment:
What does "generic" mean?

Reply:
It seems to be "The schema was not found". If yes, it's better to use `ResultSchemaNotFound` for it.

Reply:
Could this `Result` be returned with binary protocol? If it's an error that is only related to the HTTP protocol, I think it would not be a good idea to expose it as a `Result`.